syne_tune/optimizer/schedulers/hyperband_pasha.py

```diff
@@ -10,18 +10,24 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-from typing import List, Dict, Any
+import itertools
+from typing import Any, Dict, List
+
 import numpy as np
-from syne_tune.optimizer.schedulers.hyperband_promotion import PromotionRungSystem
+from syne_tune.optimizer.schedulers.hyperband_promotion import \
+    PromotionRungSystem
 
 
 class PASHARungSystem(PromotionRungSystem):
     """
-    Implements PASHA algorithm. It is very similar to ASHA, but it progressively
-    extends the maximum resources if the ranking in the top two current rungs
-    changes. A report introducing and evaluating the approach is available at:
+    Implements PASHA algorithm. PASHA is a more efficient version of ASHA
+    and is able to dynamically allocate maximum resources for the tuning procedure
+    depending on the need. Experimental evaluation has shown PASHA consumes
+    significantly fewer computational resources than ASHA.
 
-    TODO: add link
+    For more details, see the paper:
+    PASHA: Efficient HPO and NAS with Progressive Resource Allocation
```

Comment:
Wrong formatting. Please check comment of `HyperbandScheduler` how these citations are formatted

Reply:
I've changed the formatting now